docs/Reference.md

```diff
@@ -1127,6 +1128,18 @@ Example:
 
 `<WebView textZoom={100} />`
 
+### `autoManageStatusBarEnabled`
```

Comment:
Perhaps the flag should be called something like autoStatusBarStyle instead, since it references the `setStatusBarStyle` method.